lib/utils.js

```diff
@@ -766,3 +772,32 @@ function handleCustomlevelNamesOpts (cLevels) {
     return {}
   }
 }
+
+/**
+ * Translates all conditional blocks from within the messageFormat. Translates any matching
+ * {if key}{key}{end} statements and returns everything between if and else blocks if the key provided
+ * was found in log.
+ *
+ * @param {string} messageFormat A format string or function that defines how the
+ *  logged message should be conditionally formatted, e.g. `'{if level}{level}{end} - {if req.id}{req.id}{end}'`.
+ * @param {object} log The log object to be modified.
+ *
+ * @returns {string} The parsed messageFormat.
+ */
+function parseMessageFormat (messageFormat, log) {
```

Comment:
I think this would be a better name:

```suggestion
function interpretConditionals (messageFormat, log) {
```

Reply:
Renamed accordingly.